chromium_src/chrome/browser/ui/tabs/tab_features.cc

```diff
@@ -5,20 +5,8 @@
 
 #include "chrome/browser/ui/tabs/public/tab_features.h"
 
-#include "brave/browser/ui/side_panel/brave_side_panel_utils.h"
```

Comment:
I don't know if we should have been able to include this here :shrug:

Reply:
Yes, that does seem problematic and there must be a corresponding dep that should be removed. We're defeating the purpose here if we have a dep on browser/ui from this target

Reply:
I'm not sure how this passed `gn check`, I don't see any existing `//brave/browser/ui` dependency (which is good), but without that it should fail.

Reply:
I thought we had made this work at some point, but only checkdeps runs for chromium_src and not gn check